components/alert/Badge/badge.css

```diff
@@ -0,0 +1,58 @@
+.avatar-badge {
+    border-radius: 50%;
+    width: 4rem;
+    height: 4rem;
+    margin: 0 10px;
+}
+
+.badge {
+    position: relative;
+    width: max-content;
+    min-width: 2.5rem;
+    font-size: 2rem;
+    margin: 0 1rem;
+}
+
+.badge>div {
+    bottom: 5px;
+    right: 10px;
+    position: absolute;
+    height: 1rem;
+    width: 1rem;
+    border-radius: 50%;
+    border: 2px solid var(--color-white);
+    background-color: var(--primary-color);
+}
+
+.badge .online {
+    background-color: var(--success-color);
+}
+
+.badge .hold {
+    background-color: var(--hold-color);
+}
+
+.badge .offline {
+    background-color: var(--color-white);
+    border: 3px solid var(--warning-color);
+}
+
+.badge .away {
+    background-color: var(--away-color);
+
+}
+
+.badge .badge-counts {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
Just a suggestion. You can extract this in a utility class

Reply:
okay.